README.md

```diff
@@ -1,8 +1,55 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+Welcome to Mutual-Aid-Project! This site is for organizing mutual aid efforts within your community. 
 
 ## Setup
 
-Environmental variables:  
+Before starting, please reach out to Megan at info@townhallproject.com for environmental variables. 
```

Comment:
While this is needed to be able to run the project locally, which is a pretty key part of development, it wouldn't necessarily have to be the first step someone takes. I might move this to after they have forked, cloned, and ran npm i. Additionally, I would probably either link to the .env template or include it right after this.

Reply:
Added the .env template at the bottom of the read me. Moved after forking and cloning.